lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -625,6 +598,56 @@ def validateFASTA(self, ctx, params):
         # return the results
         return [out]
 
+    def calculate_fq_stats(self, reads_object, file_path):
+        eautils = kb_ea_utils(self.callback_url)
```

Comment:
The ea utils client is installed so it always points to dev. This is bad practice - it should be parameterized so it can be easily changed when this module is out of development mode (or put in the deploy file).